From: Volker Krause <vkrause@kde.org>
Date: Sat, 05 Sep 2015 14:19:02 +0000
Subject: Don't localize the date here, this is part of the IMAP protocol.
X-Git-Url: http://quickgit.kde.org/?p=kimap.git&a=commitdiff&h=5baa874efeee6cdb28995037ed9184a7fb822d36
---
Don't localize the date here, this is part of the IMAP protocol.

Fix by Antonio Rojas.

BUG: 352223
---


--- a/src/appendjob.cpp
+++ b/src/appendjob.cpp
@@ -124,7 +124,7 @@
 
     if (!d->internalDate.isNull()) {
         const QDateTime utcDateTime = d->internalDate.toUTC();
-        parameters += " \"" + utcDateTime.toString(QString::fromAscii("dd-MMM-yyyy hh:mm:ss")).toLatin1() + " +0000" + '\"';
+        parameters += " \"" + QLocale::c().toString(utcDateTime, QString::fromAscii("dd-MMM-yyyy hh:mm:ss")).toLatin1() + " +0000" + '\"';
     }
 
     parameters += " {" + QByteArray::number(d->content.size()) + '}';

